driver/src/test/java/org/neo4j/driver/internal/cluster/RediscoveryTest.java

```diff
@@ -412,16 +417,20 @@ void shouldNotLogWhenSingleRetryAttemptFails()
     }
 
     @Test
-    void shouldNotResolveToIPs()
+    void shouldNotResolveToIPs() throws UnknownHostException
```

Comment:
maybe we should rename this test :)

Reply:
Good catch, thank you! I've updated the test name.